wagtail/snippets/tests.py

```diff
@@ -300,6 +318,25 @@ def test_edit_with_tags(self):
             list(snippet.tags.order_by('name')),
             expected_tags)
 
+    def test_after_edit_snippet_hook(self):
+        advert = Advert.objects.create(
+            url='http://www.example.com/',
+            text='Test hook',
+        )
+
+        def hook_func(request, instance):
+            self.assertIsInstance(request, HttpRequest)
+            self.assertQuerysetEqual(instance, ["<Advert: Test hook>"])
+            return HttpResponse("Overridden!")
+
+        with self.register_hook('after_delete_snippet', hook_func):
```

Comment:
this tests the hook deletion, rather than edit :)

Reply:
LOL This is why we do code review. Copy pasta strikes again. Will adjust in the next commit. Thanks!